### ISSUE TITLE
How to mention a user

### ISSUE THREAD

**Issue by shrx** [Rivalo/discord-cli#29]
If I type `@user name` or `@"user name"` the user is not mentioned. How can I correctly mention a user?

**Comment by RickvanLoo** [comment]
You are right, the mention parser doesn't like spaces inside usernames. This can take some time to fix, since I'm currently using regex to get all words after `@`. It's becoming more difficult when it can be multiple words...
